bower.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "sanctuary",
+  "version": "0.6.0",
+  "description": "Refuge from unsafe JavaScript",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/plaid/sanctuary.git"
+  },
+  "main": "index.js",
+  "moduleType": [
+    "globals",
+    "node"
+  ],
+  "keywords": [
+    "adt",
+    "either",
+    "functional",
+    "maybe",
+    "ramda",
+    "safe",
+    "safety",
+    "types"
+  ],
+  "dependencies": {
+    "ramda": "0.17.x"
+  },
+  "ignore": [
+    "/.git/",
+    "/bower_components/",
+    "/coverage/",
+    "/node_modules/",
+    "/scripts/",
+    "/test/",
+    "/.*",
+    "/CONTRIBUTING.md",
+    "/Makefile",
+    "/circle.yml",
+    "/package.json"
+  ]
```

Comment:
Given that I've been unable to determine which [paths Bower ignores by default][1], I've listed all the applicable paths explicitly (including paths such as `"/bower_components/"` which are surely ignored regardless).


[1]: http://stackoverflow.com/q/22598184/312785